kubectl-minio/cmd/tenant-create.go

```diff
@@ -20,27 +20,30 @@ package cmd
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"io"
+	"strconv"
+	"strings"
 
 	"github.com/minio/kubectl-minio/cmd/helpers"
 	"github.com/minio/kubectl-minio/cmd/resources"
 	"github.com/minio/minio/pkg/color"
+	miniov1 "github.com/minio/operator/pkg/apis/minio.min.io/v1"
+	operatorv1 "github.com/minio/operator/pkg/client/clientset/versioned"
+	"github.com/minio/operator/pkg/resources/services"
+	"github.com/spf13/cobra"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"sigs.k8s.io/yaml"
-
-	miniov1 "github.com/minio/operator/pkg/apis/minio.min.io/v1"
-	operatorv1 "github.com/minio/operator/pkg/client/clientset/versioned"
-	"github.com/spf13/cobra"
 )
 
 const (
 	createDesc = `
 'create' command creates a new MinIO tenant`
-	createExample       = `  kubectl minio tenant create --name tenant1 --servers 4 --volumes 16 --capacity 16Ti --namespace tenant1-ns`
+	createExample       = `  kubectl minio tenant create tenant1 --servers 4 --volumes 16 --capacity 16Ti --namespace tenant1-ns`
 	tenantSecretSuffix  = "-creds-secret"
 	consoleSecretSuffix = "-console-secret"
 )
```

Comment:
Nit for User Experience: https://godoc.org/github.com/spf13/cobra#Command recommends setting `Use` to be a more complete syntax example:

```sh
create <string> --servers <int> --volumes <int> --capacity <str> --namespace <str>
```

This way users don't have to trial-and-error. I know the example is there, but I can see a user assuming that a subset of flags or arguments are OK (or assuming flags are optional, which is reasonable).

Reply:
If there are no objections I'd recommend expanding the `use` variable for each command that takes arguments (so tenant create, expand, and upgrade)